custom_components/ha_washdata/const.py

```
@@ -487,6 +487,13 @@ class TerminationReason(StrEnum):
 MATCH_DTW_RESAMPLE_N = 200         # common grid length for "scaled"/"ddtw" DTW
 MATCH_DDTW_DIST_SCALE = 30.0       # half-saturation for derivative-DTW distance
 MATCH_DTW_ENSEMBLE_W = 0.7         # weight on L1 vs DDTW in "ensemble" mode
+# Envelope alignment grid cap: maximum number of time-grid points used by
+# compute_envelope_worker. The DTW cost matrix is (n+1)×(m+1)×8 B float64;
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)